x-pack/test/saved_object_api_integration/common/suites/import.ts

```diff
@@ -37,33 +48,60 @@ const NEW_ATTRIBUTE_VAL = `New attribute value ${Date.now()}`;
 //  * id: conflict_3
 //  * id: conflict_4a, originId: conflict_4
 // using the seven conflict test case objects below, we can exercise various permutations of exact/inexact/ambiguous conflict scenarios
-const CID = 'conflict_';
+const { HIDDEN, ...REMAINING_CASES } = CASES;
 export const TEST_CASES: Record<string, ImportTestCase> = Object.freeze({
-  ...CASES,
-  CONFLICT_1_OBJ: Object.freeze({ type: 'sharedtype', id: `${CID}1` }),
-  CONFLICT_1A_OBJ: Object.freeze({ type: 'sharedtype', id: `${CID}1a`, originId: `${CID}1` }),
-  CONFLICT_1B_OBJ: Object.freeze({ type: 'sharedtype', id: `${CID}1b`, originId: `${CID}1` }),
-  CONFLICT_2C_OBJ: Object.freeze({ type: 'sharedtype', id: `${CID}2c`, originId: `${CID}2` }),
-  CONFLICT_2D_OBJ: Object.freeze({ type: 'sharedtype', id: `${CID}2d`, originId: `${CID}2` }),
+  ...REMAINING_CASES,
+  CONFLICT_1_OBJ: Object.freeze({ type: 'sharedtype', id: `conflict_1` }),
+  CONFLICT_1A_OBJ: Object.freeze({ type: 'sharedtype', id: `conflict_1a`, originId: `conflict_1` }),
+  CONFLICT_1B_OBJ: Object.freeze({ type: 'sharedtype', id: `conflict_1b`, originId: `conflict_1` }),
+  CONFLICT_2A_OBJ: Object.freeze({ type: 'sharedtype', id: `conflict_2a`, originId: `conflict_2` }),
+  CONFLICT_2C_OBJ: Object.freeze({ type: 'sharedtype', id: `conflict_2c`, originId: `conflict_2` }),
+  CONFLICT_2D_OBJ: Object.freeze({ type: 'sharedtype', id: `conflict_2d`, originId: `conflict_2` }),
   CONFLICT_3A_OBJ: Object.freeze({
     type: 'sharedtype',
-    id: `${CID}3a`,
-    originId: `${CID}3`,
-    expectedNewId: `${CID}3`,
+    id: `conflict_3a`,
+    originId: `conflict_3`,
+    expectedNewId: `conflict_3`,
+  }),
+  CONFLICT_4_OBJ: Object.freeze({
+    type: 'sharedtype',
+    id: `conflict_4`,
+    expectedNewId: `conflict_4a`,
   }),
-  CONFLICT_4_OBJ: Object.freeze({ type: 'sharedtype', id: `${CID}4`, expectedNewId: `${CID}4a` }),
   NEW_SINGLE_NAMESPACE_OBJ: Object.freeze({ type: 'isolatedtype', id: 'new-isolatedtype-id' }),
   NEW_MULTI_NAMESPACE_OBJ: Object.freeze({ type: 'sharedtype', id: 'new-sharedtype-id' }),
   NEW_NAMESPACE_AGNOSTIC_OBJ: Object.freeze({ type: 'globaltype', id: 'new-globaltype-id' }),
 });
+export const SPECIAL_TEST_CASES: Record<string, ImportTestCase> = Object.freeze({
+  HIDDEN,
```

Comment:
We already treated the "HIDDEN" test case as a special case before, I'm just making that clearer now.